Sources/NIOSSL/TLSConfiguration.swift

```diff
@@ -84,6 +84,55 @@ public enum NIORenegotiationSupport {
     case always
 }
 
+/// Signature algorithms. The values are defined as in TLS 1.3
+public enum SignatureAlgorithm : UInt16 {
```

Comment:
Let’s not use an `enum` here: we can’t extend them in semver minors.

This should be a `RawRepresentable` `struct` instead.